src/Symfony/Component/Notifier/Bridge/AmazonSns/.gitattributes

```
@@ -0,0 +1,4 @@
+/Tests export-ignore
+/phpunit.xml.dist export-ignore
+/.gitignore export-ignore
+/.gitattributes export-ignore
```

```suggestion
/.gitattributes export-ignore
/.gitignore export-ignore
```

to match the other bridges, thanks.